Libraries/Components/Pressable/Pressable.js

```diff
@@ -49,6 +49,11 @@ type Props = $ReadOnly<{|
   accessibilityLiveRegion?: ?('none' | 'polite' | 'assertive'),
   accessibilityRole?: ?AccessibilityRole,
   accessibilityState?: ?AccessibilityState,
```

Comment:
Let's keep these props alpha sorted instead of placing them within the accessibility props

Reply:
Sure, I will change them.